app/build.gradle.kts

```diff
@@ -53,6 +53,16 @@ android {
             excludes += "/META-INF/{AL2.0,LGPL2.1}"
         }
     }
+
+    ksp {
+        arg("room.incremental", "true")
+        // TODO uncomment once we have the first valid version of DB
+        // arg("room.schemaLocation", "$projectDir/schemas")
+    }
+
+    hilt {
+        enableAggregatingTask = true
+    }
```

Comment:
There was a warning that said "The following options were not recognized by any annotation processor: ...". This fixed it.